public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -87,17 +90,113 @@ a#i18n-attribute
 +makeExample('cb-i18n/ts/app/app.component.html', 'i18n-attribute-meaning', 'app/app.component.html')(format=".")
 
 :marked
-  While all appearance of a message with the _same_ meaning should have the _same_ translation, 
+  While all appearance of a message with the _same_ meaning have the _same_ translation, 
   a message with *different meanings* could have different translations.
   The Angular extraction tool preserves both the _meaning_ and the _description_ in the translation source file
   to facilitiate contextually-specific translations.
 
+  Also, if you want to translate some content without having to create a new element, you can use a `<ng-container>`
+  which doesn't get renderered in the DOM:
```

Comment:
Wasn't clear to me at all. I think I can fix that. But why care if the tag is rendered or not? Why not just use a span?